src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -833,12 +549,7 @@ public boolean contains(String id) {
      * @see <a target="_blank" href="https://docs.cloudant.com/document.html#delete">Documents -
      * delete</a>
      */
-    public com.cloudant.client.api.model.Response remove(Object object) {
-        Response couchDbResponse = db.remove(object);
-        com.cloudant.client.api.model.Response response = new com.cloudant.client.api.model
-                .Response(couchDbResponse);
-        return response;
-    }
+    com.cloudant.client.api.model.Response remove(Object object);
```

Comment:
If possible, we should remove the namespacing and just import response directly.